string robot_name
-------
bool success
int32 mir_code